PKGBUILD

```diff
@@ -9,11 +9,12 @@ pkgver=2021.01
 pkgrel=1
 _srcname=u-boot-${pkgver}
 pkgdesc="U-Boot for Orange Pi 4"
-arch=('aarch64')
+arch=('x86_64' 'aarch64')
 url='http://www.denx.de/wiki/U-Boot/WebHome'
 license=('GPL')
 backup=('boot/boot.txt' 'boot/boot.scr')
-makedepends=('bc' 'git')
+makedepends=('bc' 'git' 'dtc')
+options=(!distcc !strip !ccache)
```

Comment:
I don't see any reason to override the user's configs here.

Reply:
strip breaks cross compiling and provides no benefit outside of binary sizes afaik; Will remove, but advising my reason for admission.
distcc and ccache is a uncaught mistake, a leftover from the testing environment; will remove.